adyenv6b2ccheckoutaddonv2/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen-checkout/src/components/thank-you-page/ThankYouPage.tsx

```diff
@@ -1,17 +1,101 @@
 import React from "react";
+import {translationsStore} from "../../store/translationsStore";
+import {PaymentPending} from "./PaymentPending";
+import {PaymentStatus} from "../../types/paymentStatus";
+import {PaymentSuccess} from "./PaymentSuccess";
+import {PaymentFailed} from "./PaymentFailed";
+import {PaymentRejected} from "./PaymentRejected";
+import {PaymentTimeout} from "./PaymentTimeout";
+import {PaymentStatusService} from "../../service/paymentStatusService";
+import {isNotEmpty} from "../../util/stringUtil";
 
 interface Props {
     orderCode: string
 }
 
+interface State {
+    numberOfStatusChecks: number,
+    paymentStatus: PaymentStatus
+}
+
+export class ThankYouPage extends React.Component<Props, State> {
+    private statusRequestInterval = 2 * 1000; // 2 * 1000ms
```

Comment:
Is this comment needed?

Reply:
I think it suggests time unit used in timer